opm/simulators/linalg/bda/opencl/openclBILU0.cpp

```diff
@@ -31,6 +31,12 @@
 
 #include <sstream>
 
+#if HAVE_OPENMP
+#include <thread>
+#include <omp.h>
+extern std::shared_ptr<std::thread> copyThread;
+#endif //HAVE_OPENMP
+
```

Comment:
Please explain why we mix `std::thread` and OpenMP.

The former should work without the latter.

Reply:
Yes, your are right, it should not and I fixed it now.